### PR TITLE
Fix merge conflict in https://github.com/oaeproject/3akai-ux/pull/3701

### PR DESCRIPTION
**Pull request by stuartf**

### CODE REVIEW

**Comment by stuartf** [comment]
This is just a fix of the merge conflict in #3701

File: shared/oae/css/oae.base.css [hunk 1]
```diff
@@ -464,14 +464,24 @@ ul.as-selections li.as-original input::-ms-clear {
...
+            border-radius: 3px;
+    -webkit-box-shadow: none;
+       -moz-box-shadow: none;
+            box-shadow: none;
     font-family: inherit;
     line-height: 30px;
     margin-bottom: 3px;
     margin-right: 5px;
-    max-width: 400px;
+    max-width: 300px;
```
Comment (nicolaasmatthijs) [inline shared/oae/css/oae.base.css]: If the span has a `max-width` and a `width: auto`, I don't think the `li` needs them

File: shared/oae/css/oae.base.css [hunk 2]
```diff
@@ -464,14 +464,24 @@ ul.as-selections li.as-original input::-ms-clear {
 }
 
 ul.as-selections li.as-selection-item {
-    border-radius: 3px;
-    box-shadow: none;
+    -webkit-border-radius: 3px;
```
Comment (nicolaasmatthijs) [inline shared/oae/css/oae.base.css]: We don't do vendor prefixes for `border-radius`, so these can be removed

File: shared/oae/css/oae.base.css [hunk 3]
```diff
@@ -464,14 +464,24 @@ ul.as-selections li.as-original input::-ms-clear {
...
+       -moz-border-radius: 3px;
+            border-radius: 3px;
+    -webkit-box-shadow: none;
```
Comment (nicolaasmatthijs) [inline shared/oae/css/oae.base.css]: I don't think we need to do vendor prefixes for `box-shadow` either. You can remove these + any other `box-shadow` vendor prefixes we have

**Comment by nicolaasmatthijs** [comment]
Finished review. Back to @stuartf for follow-up

**Comment by stuartf** [comment]
fixed review issues

**Comment by nicolaasmatthijs** [comment]
Looks good. Merged.

**Pull request merged**